src/index.ts

```diff
@@ -501,7 +511,7 @@ function startWorkerThread<R, T extends AnyAsyncFn<R>>(
       : workerPathUrl,
     {
       eval: useEval,
-      workerData: { workerPort },
+      workerData: { workerPort, sharedBuffer, useBuffer },
```

Comment:
```suggestion
      workerData: useBuffer ? { sharedBuffer } : { workerPort },
```

Reply:
No, we need the buffer anyway for the `Atomics.wait`
Although we could indeed infer the `useBuffer` on the other side but I don't know what the relative performance cost is between checking a boolean on an existing object or passing one more property through an object.

I personally prefer the stability of knowing that the properties are here anyway and simplifying the type definition.

Reply:
```js
const useBuffer = 'growable' in SharedArrayBuffer.prototype
```

The above can be used outside, no need to pass it at all.